DrinkManagerConsole/SearchDrinkConsoleUi.cs

```diff
@@ -79,5 +82,29 @@ public static void StartSearch(List<Drink> drinksList, SearchCriterion searchCri
                 }
             } while (continueSearch);
         }
+
+        public static void AddMoreDrinksFromFile(List<Drink> drinksList)
+        {
+            Console.WriteLine("Please provide full path to the source file");
+            var path = Console.ReadLine();
+            var loader = new DrinkLoader();
+            try
+            {
+                loader.AddDrinksFromFile(drinksList, path);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($"\n{e.Message}");
+                // This should be replaced with a method maybe like "WaitForAnyKey(string message)"
+                Console.WriteLine("\nPress any key to go back to the main menu.");
+                Console.ReadKey();
+                return;
+            }
+
+            Console.WriteLine("New drinks added!");
+            // This should be replaced with a method maybe like "WaitForAnyKey(string message)"
+            Console.WriteLine("\nPress any key to go back to the main menu.");
+            Console.ReadKey();
```

Comment:
I think that is a good idea to create such a method and I would use it later on. The code is fine